packages/python-pulp-container/python-pulp-container.spec

```diff
@@ -28,14 +28,12 @@ Summary:        %{summary}
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-ecdsa >= 0.14
 Conflicts:      %{?scl_prefix}python%{python3_pkgversion}-ecdsa >= 0.15
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore < 3.20
-Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore >= 3.17.0
+Requires:       %{?scl_prefix}python%{python3_pkgversion}-pulpcore >= 3.18.0
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-pyjwkest >= 1.4.0
 Conflicts:      %{?scl_prefix}python%{python3_pkgversion}-pyjwkest >= 1.5
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-pyjwt >= 1.7.1
 Conflicts:      %{?scl_prefix}python%{python3_pkgversion}-pyjwt >= 1.8
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-setuptools
-Requires:       %{?scl_prefix}python%{python3_pkgversion}-url-normalize >= 1.4.2
```

Comment:
I'm assuming this is replaced by the dependency on url-normalize by the pulpcore package?